FPC build test fails to install marbles02 (due to fabric-ccenv version 2.3.3)

**Describe the bug**

- FPC build error.

**To Reproduce**

1. clone FPC
2. `cd $FPC_PATH/utils/docker`
3. `make pull-dev; make run-dev`
4. `cd $FPC_PATH` in fpc-develompent-container 
5. `make` in fpc-develompent-container 
- The same error seems to occur when running CI using Github action.

**Expected behavior**

- No build error

**Log-filesScreenshots**

```
2021-09-14 07:34:55.079 UTC [chaincodeCmd] ClientWait -> INFO 001 txid [079ec5b5dd1286e959e3077e5b78d299a145f9b0f84f5e9c10331e3595bd05d8] committed with status (VALID) at 127.0.0.1:7051
2021-09-14 07:34:55.079 UTC [chaincodeCmd] chaincodeInvokeOrQuery -> INFO 002 Chaincode invoke successful. result: status:200
Querying Chaincode Encryption Key
peer.sh: /project/src/github.com/hyperledger/fabric-private-chaincode/fabric/_internal/bin/peer chaincode query -o localhost:7050 -C mychannel -n ercc -c {"Args":["QueryChaincodeEncryptionKey", "auction_test"]}
Chaincode EK (b64): LS0tLS1CRUdJTiBSU0EgUFVCTElDIEtFWS0tLS0tCk1JSUJpZ0tDQVlFQXc3NWxwUkRMdWt6cWw0eTV1VTB2dVpTelhuT255SUVPMEJLMmM5S2V6RDhIbk9vMkVXZVYKRlM0MDdUaEEvSDNyRU03QkREZkhXUTRwdFlIZkxmcjJHckMvREhiZ3RoUzJyTEgxMlFIc1R5NnhIaFBhVGIzSgpqdnJxc3ZFb0xhSUFNRTFxbXpibWw5WGdqeDV2NUcvdlZhOTYrT2VuN2p3WVdmTlN2NTZXYjRUNGthNUFRbVUxClpyT2FFdEtPNHRiSFc5bzBDY2szLzA5RFNzMkhyMFBqT0NqMlkzc1dqUXZ4bEpBajR4Y0k2cHF0ZGdoQnZENDkKY1F6cENaMDFjeS9YSFZQRWt3NjJsNk9kWGdzZXJUdFpieTVDTmdtaG9SVGNRMHh4dHNrOEVzQUEzelJZWkkvMApHK2lSVmFpeDI3dHRoTldhdEFxV0xlQi83eUpNcmlrT1BmMFFmaGNaUmV4dUFBeEo5UEgxemVKZWJsM2JGUTAxCjd1dm1Va3h4a3hYM29ZdjU2ZW1VT2c5UUtKUWNSbHRnUE5xOWVjckdvT3Jkd0JVcFFmbWdDSmRoZkVZRHRUTE0KOHhncTl0N2RzRE15MER0ZFFZOEU1dzVBcUd6RUNJeVFsMGo0eE9OMTcyUnBQcTZoZ0RpMjluYWtBVElxMGxnMQoreEZxNTYxRkNsbzFBZ01CQUFFPQotLS0tLUVORCBSU0EgUFVCTElDIEtFWS0tLS0tCg==
Error: chaincode install failed with status: 500 - failed to invoke backing implementation of 'InstallChaincode': could not build chaincode: docker build failed: docker image build failed: docker build failed: Error returned from build: 1 "no required module provides package github.com/hyperledger/fabric-samples/chaincode/marbles02/go: go.mod file not found in current directory or any parent directory; see 'go help modules'
"
peer.sh: test failed: /project/src/github.com/hyperledger/fabric-private-chaincode/fabric/_internal/bin/peer lifecycle chaincode install /tmp/marbles02.tar.gz
deployment_test.sh: test failed: env FABRIC_CFG_PATH=/project/src/github.com/hyperledger/fabric-private-chaincode/integration/config /project/src/github.com/hyperledger/fabric-private-chaincode/integration/../fabric/bin//peer.sh lifecycle chaincode install /tmp/marbles02.tar.gz
make[1]: *** [Makefile:22: deployment_test] Error 111
make[1]: Leaving directory '/project/src/github.com/hyperledger/fabric-private-chaincode/integration'
make: *** [Makefile:22: test] Error 2
```

**Environment (please complete the following information):**
<!--
 - OS (distro+version): [e.g. Ubuntu 20.04]
 - FPC Version (tag or commit-id) [e.g. 'concept-release']
-->

- My Environment
  - Machine (VM): Microsoft Azure DCsv2 series (include Intel SGX)
  - OS: Ubuntu 20.04
  - FPC version: v1.0-rc
  - FPC mode: both (HW and SIM)

**Additional context**

- This issue was discussed in Rocket chat
  - https://chat.hyperledger.org/channel/fabric-private-chaincode?msg=pHmaXopn7Jer2Ghmk

@mbrandenburger 
Thanks! I confirmed that I could build the FPC.

However, I have discovered a new issue.
I tried the test-network deployment.
```
cd $FPC_PATH/samples/deployment/test-network
export CC_ID=echo
export CC_PATH=$FPC_PATH/samples/chaincode/echo
make build
```

I got the following error.
```
# make build
make -C /project/src/github.com/hyperledger/fabric-private-chaincode/ercc all docker
make[1]: Entering directory '/project/src/github.com/hyperledger/fabric-private-chaincode/ercc'
# hard to list explicitly, so just leave empty target,
# which forces ecc to always be built
# ERCC's binary is created here.
# The binary is then referenced by the fpc-peer cli, inserted in the ERCC package,
# and eventually run by the external launcher.
go  build -tags WITH_PDO_CRYPTO -o ercc main.go
go  test -tags WITH_PDO_CRYPTO -v -race -covermode=atomic -coverprofile=coverage.out ./...
?       github.com/hyperledger/fabric-private-chaincode/ercc    [no test files]
?       github.com/hyperledger/fabric-private-chaincode/ercc/attestation        [no test files]
2021-09-16 01:13:04.832 UTC [cgo] 0 -> INFO 001 Initializing logger
=== RUN   TestRegisterEnclave
--- PASS: TestRegisterEnclave (0.00s)
=== RUN   TestQueryListEnclaveCredentials
--- PASS: TestQueryListEnclaveCredentials (0.00s)
=== RUN   TestQueryEnclaveCredentials
--- PASS: TestQueryEnclaveCredentials (0.00s)
PASS
coverage: 44.5% of statements
ok      github.com/hyperledger/fabric-private-chaincode/ercc/registry   0.140s  coverage: 44.5% of statements
?       github.com/hyperledger/fabric-private-chaincode/ercc/registry/fakes     [no test files]
DOCKER_BUILDKIT=1 docker  build  \
        --build-arg FPC_VERSION=main \
        -t fpc/ercc:main \
        .
[+] Building 0.3s (4/4) FINISHED
 => [internal] load build definition from Dockerfile                                                                                    0.0s
 => => transferring dockerfile: 38B                                                                                                     0.0s
 => [internal] load .dockerignore                                                                                                       0.1s
 => => transferring context: 2B                                                                                                         0.0s
 => ERROR [internal] load metadata for docker.io/hyperledger/fabric-private-chaincode-ccenv:main                                        0.2s
 => [auth] hyperledger/fabric-private-chaincode-ccenv:pull token for registry-1.docker.io                                               0.0s
------
 > [internal] load metadata for docker.io/hyperledger/fabric-private-chaincode-ccenv:main:
------
failed to solve with frontend dockerfile.v0: failed to create LLB definition: pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed
make[1]: *** [Makefile:36: docker] Error 1
make[1]: Leaving directory '/project/src/github.com/hyperledger/fabric-private-chaincode/ercc'
make: *** [Makefile:19: ercc-container] Error 2
```

--- a/fabric/Makefile
+++ b/fabric/Makefile
@@ -20,7 +20,7 @@ fetch: $(FABS_FETCHED)
 
 $(FABS_FETCHED):
 	mkdir -p $(FABS) && cd $(FABS) \
-	&& curl -sSL https://bit.ly/2ysbOFE | bash -s -- $(FABRIC_VERSION) 1.4.9 -s -d \
+	&& curl -sSL https://bit.ly/2ysbOFE | bash -s -- $(FABRIC_VERSION) 1.4.9 -s \
 	&& touch .fetched
 
 

